From 33318a4642ebaad81b57d7d0b1b4272c267622d9 Mon Sep 17 00:00:00 2001
From: "Chen, Gang G" <gang.g.chen@intel.com>
Date: Mon, 15 Apr 2024 09:37:49 +0800
Subject: [PATCH 47/54] Pass firmware parameters to OS

Androidboot paramaters will be passed to bootconfig,
Others pass to kernel cmdline

Tracked-On: OAM-117279
Signed-off-by: Chen, Gang G <gang.g.chen@intel.com>
---
 kernelflinger.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernelflinger.c b/kernelflinger.c
index e5a1d2b..ec08952 100644
--- a/kernelflinger.c
+++ b/kernelflinger.c
@@ -1238,7 +1238,7 @@ static EFI_STATUS load_image(VOID *bootimage, VOID *vendorbootimage, UINT8 boot_
 	ret = android_image_start_buffer(g_parent_image, bootimage, vendorbootimage,
 					boot_target, boot_state, NULL,
 					vb_data,
-					NULL);
+					cmd_buf);
 	if (EFI_ERROR(ret))
 		efi_perror(ret, L"Couldn't load Boot image");
 
-- 
2.25.1

